### ISSUE TITLE
Documented defaults for `import/order` groups are inaccurate

### ISSUE THREAD

**Issue by mpalmer685** [benmosher/eslint-plugin-import#601]
From the documentation for the `import/order` rule's "groups" option:

> The default value is ["builtin", "external", "internal", "parent", "sibling", "index"]

However, the rule's source file contains instead

``` js
const defaultGroups = ['builtin', 'external', 'parent', 'sibling', 'index']
```

where `internal` is missing. Not sure which of the two should be changed (I assume the source file), but this sent me on a wild goose chase this afternoon.

**Comment by jfmengels** [comment]
Good catch @mpalmer685!

Yeah, `internal` should be added to the source. Will try to fix this tonight, sorry for the inconvenience!
